README

```diff
@@ -39,6 +39,7 @@ Disadvantage:
   * space inefficency for small files: UFFS use at least one
    'block'(the minial erase unit for NAND flash, e.g. 16K ) for a file.
   * maximum supported blocks: 2^16 = 65535
+  * maximum supported partition size: 2^32-1 = 4294967295 bytes (in reality 2^32 - the size of one block)
```

Comment:
If this limitation is because of `unsigned long` usage, then this is only true  when compiling for 32-bit or LLP64 systems. On Unix-like systems (which is LP64) this `unsigned long` is 64-bit value and removes this limitation. If we want to push those changes to upstream then it's probably good idea to clarify this limitation

Reply:
It should make more sense now